src/helper.cc

```diff
@@ -1273,7 +1264,7 @@ StatefulEnqueue(statefulhelper * hlp, Helper::Xaction * r)
     /* do this first so idle=N has a chance to grow the child pool before it hits critical. */
     if (hlp->childs.needNew() > 0) {
         debugs(84, DBG_CRITICAL, "Starting new " << hlp->id_name << " helpers...");
```

Comment:
Please take advantage of the chance to drop this overly verbose `debugs()` - better info is displayed by the `openSessions()` methods.

```suggestion
```

Reply:
I am not sure whether the openSession() message is better, but there are few differences between them:
* this message level is 0 while the openSession()'s is 1
* openSession() prints nothing is hlp->cmdline is nil
* hlp->id_name is a helper category, while openSession() prints program name instead.

Reply:
@eduard-bagdasaryan, you are right, but this line is 95% noise, and we will be better off without it. We can improve openSession() debugging later as needed. I will merge the suggested change from GitHub.